src/SnackbarContainer.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import clsx from 'clsx';
-import { styled } from '@material-ui/core/styles';
+import { styled } from '@mui/styles';
```

Comment:
this is where the code is depending on `styled`, which is no longer exported by `@mui/material`

Reply:
Change it to `import { styled } from '@mui/material';`